devlab/jenkins/run_functional_tests.sh

```diff
@@ -43,12 +39,11 @@ pip install pip==6.1.1
 pip install --allow-all-external -r requirements.txt -r test-requirements.txt
 
 echo "Preparing lab"
-
 pushd devlab
 echo 'Removing old VMs if exist, first try with vagrant...'
 vagrant destroy --force
-echo 'then with vboxmanage...'
-for v in `vboxmanage list vms | awk '{print $1}'`; do vboxmanage unregistervm --delete $v; done
+echo 'then unregister VMs...'
+vboxmanage list vms | awk -F"{" '{print $2}' | awk -F"}" '{print $1}' | xargs -I {} vboxmanage unregistervm --delete {}
```

Comment:
Please, don't remove for and use sed:

for vm in `vboxmanage list vms | awk '{print $2}' | sed 's/^{*//;s/}*$//'`; do vboxmanage unregistervm --delete $vm; done